public/_redirects

```diff
@@ -7,7 +7,6 @@ https://asyncapi.io/* https://www.asyncapi.com/:splat 301!
 https://www.asyncapi.io/* https://www.asyncapi.com/:splat 301!
 
 # Docs
```

Comment:
@derberg I don't understand how to update this file, can you show me?

Reply:
no worries, so redirect is one line with 2 strings, the 1st is the list that users go to, 2nd is where we redirect them, 3 is what status code we return to the client.

so status code is always the same, `301!`, because the fact is that content changed its location permanently

example for `docs/getting-started` is:
```
/docs/getting-started /docs/tutorials/getting-started 301!
```